src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -56,29 +54,61 @@ public void start(Stage primaryStage) throws Exception {
 
         // Sign in page
 
-        this.thestage = primaryStage;            // Initialize the main stage
+        this.thestage = primaryStage;                           // Initialize the main stage
+
+        BorderPane signInPane = new BorderPane();               // Initialize panes
+        FlowPane signInLabelPane = new FlowPane();
+        HBox inputMasterBox = new HBox();
+        inputMasterBox.setSpacing(5);
+        VBox inputVBox = new VBox();
+        HBox usernameHBox = new HBox();
+        HBox passHBox = new HBox();
 
-        signInPane = new BorderPane();                      // Initialize panes
-        signInLabelPane = new FlowPane();
-        signInLabelPane.setAlignment(Pos.BOTTOM_CENTER);       // Make sure label will be in center
+        // Set Pane alignments
 
-        signInLabel = new Label("Sign-in screen");      // Initialize label
-        signInLabel.setFont(Font.font(20));                 // Set style
+        signInLabelPane.setAlignment(Pos.BOTTOM_CENTER);
+        inputMasterBox.setAlignment(Pos.CENTER);
+        inputVBox.setAlignment(Pos.CENTER);
+        usernameHBox.setAlignment(Pos.CENTER);
+        passHBox.setAlignment(Pos.CENTER);
+
+        Label signInLabel = new Label("Sign-in screen");       // Title for sign-in page
+        Label userLabel = new Label("Username:");
+        Label passLabel = new Label("Password:");
+        signInLabel.setFont(Font.font(20));                         // Set style
```

Comment:
You might want to think about storing these font sizes in variables. This might make it easier to stay consistent as you add more screens and can use the variables instead of hard-coding numbers. Or if this is only going to be used on this screen, then hard-coding might be okay.